apps/web/components/RegisterMember.tsx

```diff
@@ -264,7 +258,7 @@ export function RegisterMember({
       />
       <div className="">
         <div className="flex gap-4">
-          <div className="items-center rounded-lg bg-info px-4 py-3 text-sm font-bold text-white">
+          {/* <div className="items-center rounded-lg bg-info px-4 py-3 text-sm font-bold text-white">
```

Comment:
Remove it ?

Reply:
I did and also added Community registration cost to the CommunityPage header